src/bin/pg_autoctl/cli_common.c

```diff
@@ -301,6 +306,14 @@ cli_create_node_getopts(int argc, char **argv,
 				break;
 			}
 
+			case 'x':
```

Comment:
not sure whether this still helps usability in any way

Reply:
Yeah, it's just that both `r` and `R` are already taken, and we don't document short letters anyway. We want people to use long options so that bug reports are easy for us to figure out. It's just that `getopt_long` API requires those short letters for us to be able to process the long option names anyway...